src/pint/models/binary_piecewise.py

```diff
@@ -0,0 +1,465 @@
+"""The BT (Blandford & Teukolsky) model.
+
+See Blandford & Teukolsky 1976, ApJ, 205, 580.
+"""
+import astropy.units as u
+from astropy.table import Table
+from pint.models.parameter import (
+    MJDParameter,
+    floatParameter,
+    prefixParameter,
+    maskParameter,
+)
+import numpy as np
+from pint.toa_select import TOASelect
+from pint import GMsun, Tsun, ls
+from pint.models.parameter import floatParameter
+from pint.models.pulsar_binary import PulsarBinary
+from pint.models.stand_alone_psr_binaries.BT_model import BTmodel
+from pint.models.timing_model import MissingParameter, TimingModel
+from pint.models.stand_alone_psr_binaries.BT_piecewise import BTpiecewise
+
+
+class BinaryBTPiecewise(PulsarBinary):
+    """Model implemenring the BT model.
```

Comment:
And this docstring should also be updated

Reply:
Corrected